### PR TITLE
Implement CanNavigate function

### PR DESCRIPTION
**Pull request by neptunao**

### CODE REVIEW

File: router.go [hunk 1]
```diff
@@ -170,6 +170,15 @@ func (r *Router) Navigate(path string) {
 	}
 }
 
+// Can navigate returns true if the specified path can be navigated by the
```
Comment (albrow) [inline router.go:173]: This is just nitpicking, but would could you change the first word to `CanNavigate` to match the name of the method exactly?

I like to follow the guidelines for documenting Go code, found here: https://blog.golang.org/godoc-documenting-go-code.

**Comment by albrow** [comment]
@Neptunao this looks great. Thank you :)

Following the guidelines in [Contributing.md](https://github.com/go-humble/router/blob/master/CONTRIBUTING.md) for this project, could you please close this PR and create a PR to merge into the "develop" branch instead? Thanks!

**Comment by neptunao** [comment]
@albrow sure!

File: router.go (hunk 1 above)
Comment (neptunao) [inline router.go:173]: Sure, you are right, just forgot to run golint.

**Pull request closed without merging**